packages/react-native/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "react-native",
+  "name": "@callstack/react-native-visionos",
   "version": "1000.0.0",
-  "description": "A framework for building native apps using React",
+  "description": "React Native for VisionOS",
```

Comment:
nit: let's use lowercase `visionOS` or maybe let's change description to `React Native for Apple Vision Pro`?